### PR TITLE
add-transaction-event-custom-dimensions

### PR DESCRIPTION
**Pull request by TheNoodleMoose**
I know there are talks of a code freeze, but figured I could create the pr and possibly push this to staging. Once the code freeze is over or if the code freeze decision is changed we could merge this in.

### CODE REVIEW

File: src/app/analytics/analytics.factory.js [hunk 1]
```diff
@@ -536,7 +536,10 @@ const analyticsFactory = /* @ngInject */ function ($window, $timeout, sessionSer
               brand: 'cru',
               category: cartItem.designationType.toLowerCase(),
               variant: cartItem.frequency.toLowerCase(),
-              quantity: '1'
+              quantity: '1',
+              dimension3: cartItem.frequency.toLowerCase() === 'single' ? "one-time" : 'recurring',
```
Comment (wrandall22) [inline src/app/analytics/analytics.factory.js:540]: `'one-time'`

Comment (OzzieOrca) [reply src/app/analytics/analytics.factory.js:540]: We should setup Prettier...

Comment (TheNoodleMoose) [reply src/app/analytics/analytics.factory.js]: Alright went ahead and fixed it

**Review by OzzieOrca — APPROVED** [review]

**Comment by TheNoodleMoose** [comment]
@wrandall22 Works on staging:
Screen Shot 2021-01-14 at 10 51 29 AM
I'm going to go ahead and merge if it's good with you.

**Pull request merged**